suite-common/message-system/src/messageSystemUtils.ts

```diff
@@ -205,6 +207,19 @@ export const validateEnvironmentCompatibility = (
     );
 };
 
+export const validateGeolocationCompatibility = (
```

Comment:
nit: using `is` for boolean return value is more readable, in my opinion.
```suggestion
export const isValidateGeolocationCompatibility = (
```

Reply:
I am not sure about it, because other methods use the same naming.

Reply:
other methods are 4+ years old, so fine to use `is`

Reply:
I renamed all of them because using "is" with "validate" was grammatically incorrect... now the names follow the correct pattern, like `isCountryCodeCompatible`.